back/spec/services/content_image_service_spec.rb

```diff
@@ -0,0 +1,213 @@
+require 'rails_helper'
+
+describe ContentImageService do
+  before do
+    @text_image_1, @text_image_2 = create_list :text_image, 2
```

Comment:
Why do you use instance variables here? Why don't you use `let` to bind the values, like you do for `subclass`? Maybe because `create_list` returns an array and it is not possible to bind the array elements in a `let`?

Here is an alternative:
```
let(:text_images) { create_list :text_image, 2 }
```
and use `text_images[0]` and `text_images[1]` instead of` @text_image_1` and `@text_image_2`